spec/requests/tags_spec.rb

```diff
@@ -47,4 +47,18 @@
       expect(json["data"].first["name"]).to eq portfolio.name
     end
   end
+
+  describe "Multiple Tags" do
+    before do
+      portfolio.tag_add("multiple_tag")
```

Comment:
@lindgrenj6 can we add a spec with key, namespace and value

Reply:
@mkanoor added!